pkg/network/cloud/requests.go

```diff
@@ -0,0 +1,8 @@
+package cloud
+
+// Request ...
```

Comment:
Remove unnecessary comment :eyes:

Reply:
Actually fixed it instead :smile: 
Even if not exported, it's always helpful to have comments.